SU2_CFD/src/solver_structure.cpp

```diff
@@ -3025,7 +3025,8 @@ void CSolver::UpdateAverage(const su2double weight, const unsigned long iPoint,
   const su2double* current = node[iPoint]->GetSolution();
 
   for (unsigned short iVar = 0; iVar < nVar; iVar++) {
-    const su2double new_average = (current[iVar] - average[iVar])*weight + average[iVar];
+    // BE for everything
+    const su2double new_average = (current[iVar]*weight + average[iVar])/(1.0 + weight);
```

Comment:
If we use BE here, we should use BE everywhere (including for our hybrid variables like production).  Right now CNSSolver::UpdateAverage still uses FE.  I can add a fix to change the weight to be the "weight" for BE, which will make everything consistent again.

I don't know if this will change things much, but we should at least be consistent.

Reply:
I agree: they should all be the same.  At one point early on when I was having stability problems with the channel I changed them all to BE, just to see if it helped.  It didn't, and I don't think it should affect anything much one way or the other, but I guess I forgot to switch these back.  Anyway, we can go back to FE or add an input option.  Whatever you prefer.